### ISSUE TITLE
the developer must to write mapping,it's not convenient

### ISSUE THREAD

**Issue by xfcode-ai** [nurkiewicz/spring-data-jdbc-repository#29]

**Comment by ezgitek** [comment]
I agree (if you meant _not_ instead of _to_). But, you could use this if your variable names and column names are the same (e.g. columnName -> COLUMNNAME also works).

Just write 

```
BeanPropertyRowMapper<Entry> bprm = new BeanPropertyRowMapper<Entry>(Entry.class);
			return bprm.mapRow(rs, p); 
```
in MapRow function. BeanPropertyRowMapper is under springframework.jdbc.core.

Is there a similar way for the unmapping, except Java reflection? (I'm thinking about BeanMapper, but it is not that reliable)
